frontend/src/app/main/ui/workspace/context_menu.cljs

```diff
@@ -213,6 +213,7 @@
         single?   (= (count shapes) 1)
         do-create-artboard-from-selection #(st/emit! (dw/create-artboard-from-selection))
 
+        has-frame? (->> shapes (d/seek #(= :frame (:type %))))
         has-group? (->> shapes (d/seek #(= :group (:type %))))
         has-bool? (->> shapes (d/seek #(= :bool (:type %))))
```

Comment:
All this anonymous callbacks I think should be replaced with the named predicate defined on common.page.helpers.